app/models/label.rb

```diff
@@ -1,2 +1,3 @@
 class Label < ApplicationRecord
+  has_many :tasks
```

Comment:
I think it depends on how we want to define the Labels.
My intention was that a Label can be a separate entity. So that a Label can be created without being assigned to a task (for example a Label with the title `important`).
Let me know if it's not clear enough or if you think we should do it the other way.

Reply:
got it, I got confsed with the List